### ISSUE TITLE
Summit not recognizing CUDA

### ISSUE THREAD

**Issue by Adriandl** [inspiremd/Model-generation#1]
A reproducible example can be found here /gpfs/alpine/world-shared/chm155/Reproducible_error.tar.gz

I followed John Chodera's instructions for installing OpenMM with CUDA on summit. I can run OpenMM simulations when I do an interactive session on summit and run a python executable. But when I submit the same executable through jsrun, the system tells me "Exception: No compatible CUDA device is available"

In other words, this call works

    python 2_mmgbsa.py -p "test" -n 1

but this does not,

    jsrun --smpiargs="none" -n 1 -a 1 -c 1 -g 1 python 2_mmgbsa.py -p "test" -n 1

**Comment by shantenujha** [comment]
I'm going to wait for the tests / experiments that @mturilli and @Adriandl proposed to do today, before escalating with @arnoldt100.

**Comment by hyoo** [comment]
would you share your python environment? I am missing some modules 
```
bash-4.2$ python 2_mmgbsa.py -p "test" -n 1
Traceback (most recent call last):
  File "2_mmgbsa.py", line 17, in <module>
    import docopt
ModuleNotFoundError: No module named 'docopt'
```

**Comment by mturilli** [comment]
@Adriandl I am afraid I also need instructions about how to setup the execution environment of `2_mmgbsa`. I started to create my own virtualenv but I am probably going through a set of hurdles that you have already overcome. I did the following:
```
module load python/3.7.0
module load py-virtualenv
module load gcc
virtualenv --python=python ve
 . ve/bin/activate
pip install docopt
pip install numpy
```
`numpy` fails to install because, likely, the provided pip is an ancient v10 and the wheel of numpy complains.

**Comment by mturilli** [comment]
OK, I managed to install my own `pip` and use that in my own virtualenv but numpy still fails to build. Please let me know how to proceed. Meanwhile, I have already created a smalls script to run your executable via RADICAL-Pilot (RP). Once I will have a working virtualenv, I should be able to try to run your executable on Summit via that script.

**Comment by Adriandl** [comment]
Hey, sorry for the delay. I'm using conda, not pip. The dependencies are in the [Readme](https://github.com/inspiremd/Model-generation/blob/master/README.md). I just installed docopt, openmm and pymbar -- I believe the rest came automatically as dependencies. 

@hyoo it looks like you're just missing docopt?

**Comment by mturilli** [comment]
Hi @Adriandl , I saw the dependencies but we would need the list of commands you used to successfully create a working environment on Summit.

**Comment by Adriandl** [comment]
@mturilli , were you able to follow the <b>Installing on Summit</b> instructions [from John Chodera here](https://github.com/inspiremd/conda-recipes-summit)? This was how I set up my conda environment.
